q2_metadata/tests/normalization/test_errors_classes.py

```diff
@@ -0,0 +1,169 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import yaml
+import unittest
+import numpy as np
+
+from q2_metadata.normalization._norm_check_rules import check_rule
+from q2_metadata.normalization._norm_errors import (
+    RuleError,
+    ExpectedError,
+    OntologyError,
+    RemapError,
+    ValidationError,
+    NormalizationError,
+    BlankError,
+    MissingError,
+    FormatError
+)
+
+
+class UserDefinedExceptions(unittest.TestCase):
```

Comment:
If you use `self.assertRaisesRegex` that will simplify a lot of these tests:

https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex

Reply:
Will look into this  - thanks!

Reply:
the user-defined exceptions are removed - and so are their unittests